lib/Mojolicious/Command/generate/app.pm

```diff
@@ -136,6 +139,12 @@ sub startup {
   # Documentation browser under "/perldoc"
   $self->plugin('PODRenderer');
 
+  # Config mechanism
+  $self->plugin('Config');
```

Comment:
The comment should be more educational, like `# Load configuration from hash returned by "my_app.conf"`.